File: BAEKJOON/for/1924.c
#include <stdio.h>

int main(void) {
    int month, day, count = 0;
    int monthArray[12] = {31,28,31,30,31,30,31,31,30,31,30,31};
    char dayArray[7][4] = {"SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"};
    
    scanf("%d %d", &month, &day);

    for(int i=1; i<month; i++) {
      count += monthArray[i-1];
    }
    count += day;
    printf("%s", dayArray[count%7]);
}

File: BAEKJOON/for/1924.py
# 2007년 1월 1일 : MON.
# N월 M일은 무슨 요일인지?


a, b = input().split()
month = int(a)
day = int(b)
count = 0

dayArray = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"]
monthArray = [31,28,31,30,31,30,31,31,30,31,30,31]

for i in range(1,month) :
  count += monthArray[i-1]

print(dayArray[(count + day)%7])
